.github/workflows/android_e2e.yml

```
@@ -0,0 +1,174 @@
+                cat metro.log
+                kill $METRO_PID 2>/dev/null || true
+                exit 1
+              fi
+              echo "Waiting... ($i/24)"
+              sleep 5
+            done
+            
+            # Androidアプリをビルド・インストール
+            echo "📱 Building and installing Android app..."
+            adb devices
+            
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)